consortium/decisions/2024.06.05-Transition_to_Apero_Foundation as_Fiscal_Sponsor/answers.md

```diff
@@ -0,0 +1,141 @@
+## Apero Foundation Questionaire
+
+**Date:** `2024.06.05`
+
+**Project/Organisation:**
+`"Dat-Ecosystem"`
+
+**Project/Organization Contacts:**
+* dat-ecosystem@protonmail.com
+
+**Applicant Representatives name and email:**
+* Alexander Cobleigh
+    * cblgh@cblgh.org
+* Alexander Praetorius
+    * a.praetorius@serapath.de
+* Diego Paez
+    * diego@geut-studio.com
+* Franz Heinzmann
+    * frando@unbiskant.org
+* Kevin Faaborg
+    * kevin@littlstar.com
+* Nina Breznik
+    * ninabreznik@gmail.com
+
+**Project/Organization website:**
+https:/dat-ecosystem.org
+
+-------------------------------
+
+## 1. Charitable purpose of the project:
+
+Dat-Ecosystem's purpose is described in our manifesto:
+https://github.com/dat-ecosystem/organization/blob/13eb30bb4270f3c1d10f2bf4e36f01f636c3e90f/MANIFESTO.md
+
```

Comment:
Dat Ecosystem Goals: 

> - Share our knowledge on building decentralized systems.
> - Publish our work under permissive, open culture, licenses.
> - Build decentralized systems that profit the general public and empower users.
> - Have dat protocols (e.g. [hypercore-protocol](https://hypercore-protocol.org/) or similar) as a foundation of our work.
> - Promote our shared goals with the public.
> - Develop solutions to shared technical problems.
> - Find and invite new members that share our goals.
> - Promote the adoption of shared technology through documentation or standardization.

Reply:
Do you think we should list those explicitly rather than linking to the Manifesto?

Reply:
I think its handy and looks good, we are also quoting ourselves, which is good.

Reply:
Alright, updated that as well. Thanks